openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -185,12 +198,25 @@ def get_plugin_info(self):
         instance = self._instance
         context = instance.context
 
+        # Set it to default Maya behaviour if it cannot be determined
+        # from instance (but it should be, by the Collector).
+
+        default_rs_include_lights = instance.context.data\
+            .get('project_settings')\
+            .get('maya')\
+            .get('RenderSettings')\
+            .get('enable_all_lights')
```

Comment:
This `.get().get().get()` feels wrong. As soon as one of the keys doesn't exist it'd be `None` and doing `.get()` would fail. So it's just as prone to error as just doing `["project_settings"]["maya"]["RenderSettings"]["enable_all_lights"]`

Also the fallback down below to the default value could thus still fall back to "None" otherwise if `enable_all_lights` wouldn't exist.

Reply:
you are right, I've just copy&pasted it from somewhere. I guess it was like that because of readability but that is questionable anyway.